src/commands/archive-card.js

```diff
@@ -164,6 +180,11 @@ var __ = function(program, output, logger, config, trello, translator) {
                     metavar: 'MATCH',
                     help: "[exact | contains] : how to match the title given",
                     required: false
+                },
+                "force": {
```

Comment:
Thanks for adding the force option. You'll need to automatically archive the card rather than triggering `promptListSelection` if `options.force` is true